tasks/website/src/linter/snapshots/cli_terminal.snap

```diff
@@ -81,5 +81,7 @@ Available positional items:
 
 Available options:
         --rules               list all the rules that are currently registered
+        --experimental-nested-config  Enables automatic loading of nested configuration files
```

Comment:
NIT: its not aligned with the other options :/

Reply:
This is bpaf's problem :-/

Reply:
> This is bpaf's problem :-/

Intentional design decision :) Always aligning to the longest option name means we can get in a situation where you have a word or two on a line (if trying to keep overall help width small-ish) or making everything super wide - not very readable either. 

Current behavior means making an attempt to align everything, but allowing long options to be ugly.

Reply:
This is actually fine once we remove the `experimental` prefix.

Reply:
I don't think it should be a long-term problem: I think this option should become a default eventually. If anything, maybe we add a `--no-nested-config` option to disable it, but we'll definitely drop the experimental part.

Reply:
> If anything, maybe we add a --no-nested-config option to disable it, 
I could not find a similar eslint flag. But there is a `root` key in the configuration file

Reply:
I think the equivalent is `--no-eslintrc` or `--no-config-lookup`:

https://eslint.org/docs/latest/use/command-line-interface#--no-eslintrc